Add depreaction notice for EPL apps support for MQTT service (#3796)
SoftwareAG/c8y-docs: content/change-logs/analytics/apama-in-c8y-20251010-Deprecation-of-MQTT-Service-support-in-EPL-Apps.md

---
date: 
title: Deprecation of MQTT Service support in EPL Apps
change_type:
  - value: change-inv-3bw8e
    label: Announcement
product_area: Analytics
component:
  - value: component-M5-cepIIS
    label: Streaming Analytics
build_artifact:
  - value: tc-KXXmo2SUR
    label: apama-in-c8y
version: 
---

> The API to support MQTT Service in EPL Apps has been deprecated. The replacement API will be provided in a future release.

Reviewer: ```suggestion
The API to support MQTT Service in EPL Apps has been deprecated. A replacement API may be provided in future.
```

I know thye chances of not doing this are low, but we usually avoid making promises about the future. And we don't have "releases" now
